app/main.py

```diff
@@ -1 +1,15 @@
-# write your code here
+def copy_file(command: str) -> str:
+    split_command = command.split()
+    cp_command, file_in_name, file_out_name = split_command
```

Comment:
If the command length will be less than 3, the error will be risen

You should check the length first, and then unpack values
Use 2 `if` stamements